test/RouteMiddlewareTest.php

```diff
@@ -265,9 +265,9 @@ public function testRoutingSuccessResultingInContainerExceptionReRaisesException
     public function routerAdapters()
     {
         return [
-          'aura'       => [ 'Zend\Expressive\Router\Aura' ],
-          'fast-route' => [ 'Zend\Expressive\Router\FastRoute' ],
-          'zf2'        => [ 'Zend\Expressive\Router\Zf2' ],
+          'aura'       => [ 'Zend\Expressive\Router\AuraRouter' ],
```

Comment:
Since this is the only place they're referenced inside this classfile, it's actually probably simpler this way; otherwise you would have had to change the import statements **and** this code. ;-)